runtime-common/stdlib/math/math-context.h

```diff
@@ -27,21 +36,16 @@ struct MathLibContext final : vk::not_copyable {
 };
 
 struct MathLibConstants final : vk::not_copyable {
-  string ONE{math_context_impl::ONE.data(), static_cast<string::size_type>(math_context_impl::ONE.size())};
-  string ZERO{math_context_impl::ZERO.data(), static_cast<string::size_type>(math_context_impl::ZERO.size())};
-  string TEN{math_context_impl::TEN.data(), static_cast<string::size_type>(math_context_impl::TEN.size())};
-  string ZERO_DOT_FIVE{math_context_impl::ZERO_DOT_FIVE.data(), static_cast<string::size_type>(math_context_impl::ZERO_DOT_FIVE.size())};
-
-  bcmath_impl_::BcNum BC_NUM_ONE{bcmath_impl_::bc_parse_number(ONE).first};
-  bcmath_impl_::BcNum BC_NUM_ZERO{bcmath_impl_::bc_parse_number(ZERO).first};
-  bcmath_impl_::BcNum BC_ZERO_DOT_FIVE{bcmath_impl_::bc_parse_number(ZERO_DOT_FIVE).first};
-
-  MathLibConstants() noexcept {
-    ONE.set_reference_counter_to(ExtraRefCnt::for_global_const);
-    ZERO.set_reference_counter_to(ExtraRefCnt::for_global_const);
-    TEN.set_reference_counter_to(ExtraRefCnt::for_global_const);
-    ZERO_DOT_FIVE.set_reference_counter_to(ExtraRefCnt::for_global_const);
-  }
+  string ONE{math_context_impl_::ONE.data(), static_cast<string::size_type>(math_context_impl_::ONE.size())};
```

Comment:
Let's move this initialization to a `cpp` file as well. There is no need to keep it here. Also, it means that we may get rid of `math_context_impl_` namespace. You may move those constants to an anonymous namespace in `cpp` file.